### PR TITLE
updated input data, added new multipliers

### PR DESCRIPTION
**Pull request by joshua-reynolds**
@blakerosenthal  Hey Blake, I took a stab at adding the bike_lane_heavy, bike_lane_medium, and bike_lane_light attributes. Can you take a look at my code updates and merge if this looks right? Thanks!

### CODE REVIEW

File: Model_Configs/network.yaml [hunk 1]
```diff
@@ -50,6 +50,9 @@ route_varcoef_bike:
     prop_link_slope_2_4: 0.371
     prop_link_slope_4_6: 1.23
     prop_link_slope_6_plus: 3.239
+    bike_lane_light:-0.15
```
Comment (blakerosenthal) [inline Model_Configs/network.yaml:53]: from mark:
```
 bike_lane_light: 0 
 bike_lane_med : 0.25
 bike_lane_heavy: 1.65
```

File: utah_bike_demand_model.py [hunk 2]
```diff
@@ -85,6 +88,7 @@ def preprocess_network(net):
             slope = net.get_link_attribute_value((a,b), 'slope')
             bike_blvd = net.get_link_attribute_value((a,b), 'bike_boulevard')
             bike_path = net.get_link_attribute_value((a,b), 'bike_path')
+            bike_lane = net.get_link_attribute_value((a,b), 'bike_lane') # New
             no_bike_lane = not net.get_link_attribute_value((a,b), 'bike_lane')
```
Comment (blakerosenthal) [inline utah_bike_demand_model.py:92]: this line can now be `no_bike_lane = not bike_lane`

File: utah_bike_demand_model.py [hunk 3]
```diff
@@ -97,6 +101,9 @@ def preprocess_network(net):
             net.set_link_attribute_value( (a,b), 'prop_link_slope_2_4', distance * ( 2.0 < slope < 4.0 ) )
             net.set_link_attribute_value( (a,b), 'prop_link_slope_4_6', distance * ( 4.0 <= slope < 6.0 ) )
             net.set_link_attribute_value( (a,b), 'prop_link_slope_6_plus', distance * ( 6.0 <= slope ) )
+            net.set_link_attribute_value( (a,b), 'bike_lane_light', distance * (bike_lane and light ) ) # New
+            net.set_link_attribute_value( (a,b), 'bike_lane_med', distance * (bike_lane and med ) ) # New
+            net.set_link_attribute_value( (a,b), 'bike_lane_heavy', distance * (bike_lane and heavy ) ) # New
             net.set_link_attribute_value( (a,b), 'no_bike_lane_light', distance * ( no_bike_lane and light ) )
```
Comment (blakerosenthal) [inline utah_bike_demand_model.py:107]: or these can just be `not bike_lane` directly, whichever you prefer.

File: utah_bike_demand_model.py (hunk 2 above)
Comment (joshua-reynolds) [reply utah_bike_demand_model.py]: went with this option

**Pull request merged**